tests/test_apidoc.py

```diff
@@ -118,6 +120,15 @@ def test_pep_0420_disabled_top_level_verify(make_app, apidoc):
     print(app._status.getvalue())
     print(app._warning.getvalue())
 
+@pytest.mark.apidoc(coderoot=(rootdir / 'root' / 'trailing_underscore'))
```

Comment:
> Yes, test for pep_0420 should be also separated.

I'll move that too then

> Should we test this with all builders? Is this case needed to be built?

In order to verify #519, I think yes, it does need to be built. But I guess that can wait until that test exists.